PendingReleaseNotes.md

```diff
@@ -2,6 +2,9 @@
 
 ## Breaking Changes
 
+- Support for `secret.adminID` and `secret.adminKey` has been deprecated and removed.
```

Comment:
I think it is only removed from the Helm charts? It is not a breaking change if users did not use Helm to deploy? That could be explained a little better.